tabs/osd.js

```diff
@@ -111,6 +111,8 @@ SYM.GLIDE_MINS = 0xD5;
 SYM.GLIDE_RANGE = 0xD4;
 SYM.FLIGHT_MINS_REMAINING = 0xDA;
 SYM.FLIGHT_DIST_REMAINING = 0x167;
+SYM.GROUND_COURSE = 0xDC;
+SYM.TRACK_ERROR = 0xDD;
```

Comment:
Please use 0xFC for course track error symbol.

Reply:
OK, missed those new icons. Added a few days ago by the looks of it ? Is there any reason they aren't included in `iNav character map.md` ? I take it 251, the mountain with an arrow sticking out of it, is terrain following ?

Reply:
Not being in the .md file is my mistake. Yes, the mountain with and arrow is terrain following.

Reply:
Yes, thank you 👍🏻